### ISSUE TITLE
Embed error

### ISSUE THREAD

**Issue by solumath** [zloutek1/MasarykBOT#57]
I don't think it's error that the command is on cooldown. I think it would be better to omit the "error" word here. What do you think?
![image](https://github.com/zloutek1/MasarykBOT/assets/56552845/3b2f4afd-57ec-463a-9019-00fd022dad53)

**Comment by zloutek1** [comment]
I agree that it might be better to either omit the Error keyword or show it as a warning.
The exception handling is in a common module so feel free to implement that change :D
